stylesheets/scroll-button.scss

```diff
@@ -0,0 +1,44 @@
+div.arrow-for-scrolling-top {
+    opacity: 0;
+    transition: 1s;
+    -webkit-transition:1s;
+    -moz-transition:1s;
+    -ms-transition:1s;
+    -o-transition:1s;
```

Comment:
We can omit the compatibility prefixes, our Webpack pipeline generates those for us!

Reply:
Thank you!
I omitted the compatibility prefixes🙇‍♂️
@JasonEtco